### PR TITLE
Manuscript Report

### PR DESCRIPTION
**Pull request by ricardd**
I added all the bits and pieces to add Manuscript Reports to csasdown. All unit tests passed on my local build.

One thing, I had to create the following two PDFs, and they do not have correct margins, and are also bigger than necessary:
- csasdown\inst\csas-style\pages\AquatManuCover.pdf
- csasdown\inst\csas-style\pages\AquatManuCouvInt.pdf

It would be best if those PDFs could be re-generated to match those for the TechReport (correct margins and smaller file size).

### CODE REVIEW

File: R/inject-bilingual-code.R [hunk 1]
```diff
@@ -121,7 +121,7 @@ inject_bilingual_code <- function(fn = get_index_filename(
     }
   }
 
-  if(doc_type == "techreport"){
+  if((doc_type == "techreport") | (doc_type == "manureport")){
```
Comment (seananderson) [inline R/inject-bilingual-code.R]: this would ideally be `||` and I guess we don't need the extra `()`
or perhaps better still, `%in%`

**Review by seananderson — COMMENTED** [review]
Looks great. There's one small style thing with the vectorized `|`. Can you remove `inst/rmarkdown/templates/manureport/skeleton/manu-report-cover-DR.docx`? I assume that was an unintentional commit? I generated those 2 PDF pages from the official Tech Report docx. I imagine you'll update those when you get your hands on the ones with fixed formatting.

File: R/inject-bilingual-code.R (hunk 1 above)
Comment (ricardd) [reply R/inject-bilingual-code.R]: Thanks @seananderson , the suggested changes are in commit f399736, and yes the Word document was unintentional.

For the PDF pages, note that they are different than for a TechReport, so perhaps you also have the official Manuscript Report docx? I used a single-page "print to pdf" from a published Manuscript Report, but creating those PDFs the same those for the TechReport are done would be best.

Comment (ricardd) [reply R/inject-bilingual-code.R]: added the PDF of the second page of the English manuscript report, generated from official template received from DFO library, commit 18b47605b1050ef02b485092f3647d5d0f4c7937

**Review by seananderson — APPROVED** [review]

**Pull request merged**